cmd/ocm/cluster/create/create.go

```diff
@@ -106,6 +128,21 @@ func run(cmd *cobra.Command, argv []string) error {
 	}
 	clusterName := argv[0]
 
+	// Parse the expiration options
+	if len(args.expirationTime) > 0 {
+		t, err := parseRFC3339(args.expirationTime, metav1.Now)
+		if err != nil {
+			return err
```

Comment:
For user friendliness, maybe wrap using Errorf to explain which flag had the problem.